docs/includes/topic_lbaasv2-plugin-overview.rst

```diff
@@ -13,7 +13,22 @@ The F5 LBaaSv2 plugin consists of an :ref:`agent <agent:home>` and a service pro
 The F5 agent manages services on your BIG-IP. When it first receives a task from the F5 driver, it starts and communicates with the BIG-IP(s) identified in the :ref:`agent configuration file`. Then, it registers its own named queue. The F5 driver assigns all ``lbaas`` tasks in the Neutron messaging queue to the agent's queue. The F5 agent makes callbacks to the F5® driver to query additional Neutron network, port, and subnet information; to allocate Neutron objects (for example, fixed IP addresses); and to report provisioning and pool status.
 
 .. image:: http://f5-openstack-lbaasv1.readthedocs.io/en/liberty/_images/f5-lbaas-architecture.png
-    :alt: F5 LBaaSv2 Plugin architecture
+   :alt: F5 LBaaSv2 Plugin architecture
 
 
+.. _start-neutron-port-note:
 
+.. important::
+
+   As of v8.3.1, the F5 LBaaSv2 driver no longer manages Neutron ports.
+
+   When you create a pool member using the command ::
```

Comment:
f5-openstack-agent.log  warning: “Member definition does not include Neutron port”
server.log warning: “Lbaas member has no associated neutron port”